contribs/application/src/main/java/org/matsim/application/analysis/LinkStats.java

```diff
@@ -19,36 +23,50 @@
 import org.matsim.core.utils.io.IOUtils;
 import picocli.CommandLine;
 
+import java.io.FileWriter;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
+import java.util.stream.Collectors;
 
-@CommandLine.Command(name = "link-stats", description = "Compute aggregated link statistics, like volume and travel time")
+@CommandLine.Command(name = "link-stats", description = "Compute aggregated link statistics, like volume, travel time and congestion")
 public class LinkStats implements MATSimAppCommand {
 
 	private static final Logger log = LogManager.getLogger(LinkStats.class);
 
-	@CommandLine.Option(names = "--events", description = "Input event file", required = true)
-	private Path events;
-
 	@CommandLine.Option(names = "--network", description = "Input network file", required = true)
 	private Path network;
 
+	@CommandLine.Option(names = "--events", description = "Input event file", required = true)
+	private Path events;
+
 	@CommandLine.Option(names = "--output", description = "Path for output csv", required = true)
 	private Path output;
 
+	@CommandLine.Option(names = "--output-congestion", description = "Path for output csv for congestion", required = true)
+	private Path outputCongestion;
```

Comment:
Perhaps it will be a good idea to use the output folder instead, then we dont need to enter the output path one by one.

Then we can have: 
Path linkStatsOutputPath = Path.of(outputFolder.toString + "/link-stats.tsv");
Path congestionAnalysisOutputPath = Path.of(outputFolder.toString + "/congestion-analysis.tsv");

Reply:
That is a reasonable suggestion and would be a bit easier to use. But one would lose the flexibility to define the file names. Personally, I prefer to have these options that need to be set explicitly.

Reply:
Okay. That makes sense.